01 - Community/People/SkepticMystic.md

```diff
@@ -10,7 +10,7 @@ publish: true
 
 - GitHub: [SkepticMystic](https://github.com/SkepticMystic/) ^github
 <!-- - Discord: `@` ^discord-->
-- Website: <https://github.com/SkepticMystic/advanced-cursors> ^website
```

Comment:
I usually ignore this one since the last plugin submitted by @SkepticMystic will usually overwrite the author URL

Reply:
Sure. I've reverted that commit. So the Website is back to the previous one (which was the advanced-cursors one)